opl/args.py

```diff
@@ -86,6 +86,10 @@ def add_ros_db_opts(parser):
     _add_generic_db_opts(parser, "ros")
 
 
+def add_hbi_db_opts(parser):
+    _add_generic_db_opts(parser, "hbi")
+
+
```

Comment:
Hello Jan. We already have `add_inventory_db_opts()`. This dual naming is annoying!